docs/source/user_guide/types.rst

```diff
@@ -215,3 +214,14 @@ You can also check the underlying PySpark data type of `Series` or schema of `Da
      |-- d: double (nullable = false)
      |-- s: string (nullable = false)
      |-- b: boolean (nullable = false)
+
+.. note::
```

Comment:
Can you add a newline after this? See https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-note